static/themes.ts

```diff
@@ -586,6 +586,15 @@ export class Themer {
         if (options.mobileViewer && newSettings.theme === 'real-dark') {
             newSettings.theme = 'darkplus';
         }
+
+        if (newSettings.theme === 'system') {
+            if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+                newSettings.theme = 'dark';
+            } else {
+                newSettings.theme = 'default';
+            }
+        }
```

Comment:
If I'm understanding correctly, this is going to clobber any user-choice with the system setting, which is not the behavior we want.

Reply:
Okay, I'll remove this nonsense in the future tense.